### DISCUSSION TITLE
Ryzen 9 8945HS Binned extremely well?

### DISCUSSION THREAD

**Discussion by ethanic17** [seerge/g-helper#2798] [General]
I'm using a 2024 G14 (8945HS) and was able to achieve a -40mW undervolt in G-Helper but am not sure if I just got lucky with silicon lottery or if the Ryzen 8000 series is just binned well (compared to older Ryzen generations). My old 2021 Blade 14 (5900HX) was only able to get a -10mW undervolt. 

Anyways was able to get much lower temperatures and lower fan speed (although not too much of a performance bump; I assume probably because ASUS designed the G14 with low acoustics in exchange for lowered performance in the first place). However battery life was still about the same, and was wondering if that's normal (I'd assume so, but I heard people undervolting Intel CPUs and were able to get an additional hour or two of runtime).

**Comment by seerge** [comment]
@ethanic17 from my experience, AMD seem to add some kind of a crash-protection to 7xxx and 8xxx series. So even if you set crazy high (or low in fact) undervolt values - it still runs fine. Just after certain value - you don't get any extra benefits.

**Reply by ethanic17** [reply]
Ah okay that makes sense actually cause I was doing testing and noticed diminishing returns past -20mW.
